### PR TITLE
add open service mesh lab to docs

### LINKED ISSUE #4200: [Docs]Add Open Service Mesh lab to docs
Current State
The recently made Open Service Mesh lab is not in the Docs.

Desired State
Let's add that lab to the docs for Open Service Mesh adapter.
https://docs.meshery.io/service-meshes/adapters/osm

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by shreyaansjain06**
Signed-off-by: shreyaansjain <shreyaansjain60@gmail.com>

**Description**
added open service mesh lab to docs for osm

This PR fixes #4200

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit aec587bec35dda256156c7427a9d5c747d3111ec at: https://61454b12d1832a84f213e937--meshery-docs-preview.netlify.app

File: docs/_adapters/osm/osm.md [hunk 1]
```diff
@@ -6,13 +6,16 @@ mesh_name: Open Service Mesh
 version: v0.5.0
 port: 10009/tcp
 project_status: stable
+lab: osm-meshery-adapter
```
Comment (warunicorn19) [inline docs/_adapters/osm/osm.md:9]:
```suggestion
lab: open-service-mesh-meshery-adapter
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 1f38bcdeb113e65f4f05916bbff8890383b1d53d at: https://6145c80acb908d38a41d129a--meshery-docs-preview.netlify.app

**Review by akashgp09 — DISMISSED** [review]

**Comment by warunicorn19** [comment]
@shreyaansjain06 sorry for the confusion, but the path `open-service-mesh-meshery-adapter` was a bit too long, so I just changed the path to `osm-meshery-adapter` on katacoda. So, if you could change it to that again, and test it out locally before you commit.

**Comment by l5io** [comment]
🚀 Preview for commit a83d4d84d747546a4649dea07b185bdc1d7e92aa at: https://614851371d7f09b55ffc97ac--meshery-docs-preview.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, Thanks @shreyaansjain06.

File: docs/_adapters/osm/osm.md (hunk 1 above)
Comment (leecalcote) [reply docs/_adapters/osm/osm.md:9]: How is this reference used? Is it material to the lab being displayed? The lab shows on https://61454b12d1832a84f213e937--meshery-docs-preview.netlify.app/service-meshes/adapters/osm now.

Comment (warunicorn19) [reply docs/_adapters/osm/osm.md:9]: @leecalcote this review is from before, the changes now are fine.

**Review by sayantan1413 — APPROVED** [review]

**Pull request merged**